src/sdk/namespace_fs.js

```diff
@@ -1952,15 +1958,17 @@ class NamespaceFS {
     async get_object_tagging(params, object_sdk) {
         const tag_set = [];
         let file_path;
-        if (params.version_id && this._is_versioning_enabled()) {
-            file_path = this._get_version_path(params.key, params.version_id);
+        let file;
+        const fs_context = this.prepare_fs_context(object_sdk);
+        // Version specific tag will return even if versioning suspended.
+        if (params.version_id) {
```

Comment:
you don't need this if else, you can just 
`file_path = await this._find_version_path(fs_context, params, true);`